### ISSUE TITLE
Change Background Colour on Hour Grid Division

### ISSUE THREAD

**Issue by rishbaranwal** [zjfjack/JZCalendarWeekView#57]
Is there any provision so that I want different colour of Hour Grid Divisions

**Comment by zjfjack** [comment]
Hi,

I am so sorry that I am too busy to reply to your issue.

The answer is no. Currently, all the horizontal and vertical grid lines and hour grid divisions are using the same `UICollectionReusableView` which is `JZGridLine`. Because this feature is not required before, I used one view to keep it simple.

Do you still need this feature? I can add it maybe in June or July because there is another quite important issue needs to be fixed currently.

Thanks

**Comment by rishbaranwal** [comment]
Thanks for reply, please add it, if possible... thanks in advance

**Comment by rishbaranwal** [comment]
Are you providing this feature, please confirm so that I can go ahead with this Library.

**Comment by zjfjack** [comment]
Yes, I will add this feature for hour grid division color only. For the specific time slots color is not supported.

**Comment by suvarnaratna** [comment]
Any update on this feature?

**Comment by suvarnaratna** [comment]
@zjfjack  Are you providing this feature, please confirm..
